cdc/sink/mysql/mysql.go

```diff
@@ -51,6 +51,9 @@ const (
 	backoffBaseDelayInMs = 500
 	// in previous/backoff retry pkg, the DefaultMaxInterval = 60 * time.Second
 	backoffMaxDelayInMs = 60 * 1000
+
+	// networkDriftDuration is used to construct a context timeout for database operations.
+	networkDriftDuration = 5 * time.Second
```

Comment:
Is 5 enough since the normal duration often exceed 8s ?

Reply:
It will be added with `writeTimeout`.